Contributing.md

```diff
@@ -13,7 +13,7 @@ As such, most web platform classes present in jsdom are implemented in two parts
 - An IDL file, such as [`Attr.idl`](https://github.com/tmpvar/jsdom/blob/master/lib/jsdom/living/attributes/Attr.idl), drawn more or less straight from the spec
 - An implementation file, such as [`Attr-impl.js`](https://github.com/tmpvar/jsdom/blob/master/lib/jsdom/living/attributes/Attr-impl.js), containing the relevant implementation logic
 
```

Comment:
We'll probably need an intro paragraph very near the top saying briefly that we use Yarn, how you can get it, and that npm will probably work similarly but can be buggy and isn't good if you're submitting a PR that updates dependencies (since we need to update yarn.lock)